airbyte-config/config-models/src/main/resources/types/StandardSyncInput.yaml

```diff
@@ -32,6 +32,10 @@ properties:
     type: array
     items:
       "$ref": StandardSyncOperation.yaml
+  webhookOperationConfigs:
+    description: The webhook operation configs belonging to this workspace. Must conform to WebhookOperationConfigs.yaml.
+    type: object
+    existingJavaType: com.fasterxml.jackson.databind.JsonNode
```

Comment:
if it needs to conform, what about referencing the java type here?

Reply:
Yeah bit of a confusing comment from me - clarifying in the next revision.